tests/system/src/testscriptapp1/java/mymod/myapp1/FXMLScriptTest.java

```diff
@@ -0,0 +1,303 @@
+import java.util.TreeMap;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import javax.script.Bindings;
+import javax.script.ScriptContext;
+
+import static myapp1.Constants.*;
+import pseudoScriptEngine.InvocationInfos;
+import pseudoScriptEngine.RgfPseudoScriptEngine;
+
+/**
+ * Modular test application for testing FXML.
+ * This is launched by ModuleLauncherTest. To only run the ModuleLauncherTest:
+ * <code>./gradlew -PFULL_TEST=true :systemTests:test --tests ModuleLauncherTest</code>
+ */
+public class FXMLScriptTest extends Application {
+
+    static boolean bDebug = false; // true; // display invocation list
+
+    /** Runs the application and invokes the tests.
+     *  @param args the command line arguments, if any given the RgfPseudoScriptEngine invocation logs get displayed
+     *              which are used in the asserCorrectInvocations() method
+     */
+    public static void main (String args[]) {
+        try {
+            // any argument will cause the bDebug flag to be set to true
+            if (args.length>0) {
+               bDebug = true;
+            }
+            new FXMLScriptTest().launch();
+            // for debugging, allows to study invocation logs in detail
+            if (bDebug) { dumpEvalInformation(); }
+            assertCorrectInvocations();
+        } catch (AssertionError ex) {
+            System.err.println("ASSERTION ERROR: caught unexpected exception: " + ex);
+            ex.printStackTrace(System.err);
+            System.exit(ERROR_ASSERTION_FAILURE);
+        } catch (Error | Exception ex) {
+            System.err.println("ERROR: caught unexpected exception: " + ex);
+            ex.printStackTrace(System.err);
+            System.exit(ERROR_UNEXPECTED_EXCEPTION);
+        }
+        System.exit(ERROR_NONE);    // not in stop() method as we need to run the assertions first
+    }
+
+    @Override
+    public void start (Stage mainStage) {
+        URL fxmlUrl     = null;
+        Parent rootNode = null;
+        Scene  scene    = null;
+        Button btn      = null;
+        try
+        {
+             fxmlUrl    = Util.getURL(FXMLScriptTest.class, "demo_01");
+             rootNode   = FXMLLoader.load(fxmlUrl);
+             scene      = new Scene(rootNode);
+             btn        = (Button) scene.lookup("#idButton");
+        }
+        catch (Exception ioe)
+        {
+            ioe.printStackTrace();
+            System.exit(-1);
+        }
+            // fire three events on the button
+        btn.fire();
+        btn.fireEvent( new ActionEvent() );
+        btn.fireEvent( new MouseEvent( MouseEvent.MOUSE_CLICKED,
```

Comment:
For the previous two, please remove the spaces after the `(` and before the `)`.